### PR TITLE
Feature 16/team member select Closes #16

### LINKED ISSUE #16: FE > Create team member select component (S)
- [x] Component should be a dialog that looks similar to: https://codepen.io/chuckrector/pen/qKMBre

- [x] Component should have a property to allow for multi-select. 

- [x] Search field should initiate the result list...search by name only for now.

### PR DESCRIPTION
**Pull request by jahanner**

### CODE REVIEW

File: web-ui/src/components/team-member/TeamMemberSelect.jsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+import "./TeamMemberSelect.css";
+
+const TeamMemberSelect = (props) => {
+  const { teamMembers, onChange, singleSelect = false } = props;
+  const [filteredTeamMembers, setFilteredTeamMembers] = useState(teamMembers);
+
+  const filterTeamMembers = (e) => {
+    let searchInput = e.target.value;
+    let filtered = teamMembers.filter((member) => {
+      return member.name.includes(searchInput);
```
Comment (mkimberlin) [inline web-ui/src/components/team-member/TeamMemberSelect.jsx:13]: can we be case insensitive?

File: web-ui/src/components/team-member/TeamMemberSelect.jsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
+import React, { useState } from "react";
+import Avatar from "@material-ui/core/Avatar";
+
+import "./TeamMemberSelect.css";
+
+const TeamMemberSelect = (props) => {
+  const { teamMembers, onChange, singleSelect = false } = props;
+  const [filteredTeamMembers, setFilteredTeamMembers] = useState(teamMembers);
```
Comment (chrismckiernan) [inline web-ui/src/components/team-member/TeamMemberSelect.jsx:8]: How will this change due to our context work? Not saying at all that you need or should do that now.

File: web-ui/src/components/team-member/TeamMemberSelect.jsx [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+  const filterTeamMembers = (e) => {
+    let searchInput = e.target.value;
+    let filtered = teamMembers.filter((member) => {
+      return member.name.includes(searchInput);
+    });
+    setFilteredTeamMembers(filtered);
+  };
+
+  const selectMultipleTeamMembers = (member) => {
+    member.selected = !member.selected;
```
Comment (jorwalk) [inline web-ui/src/components/team-member/TeamMemberSelect.jsx:19]: What is the logic and purpose for this line?

File: web-ui/src/components/team-member/TeamMemberSelect.test.js [hunk 4]
```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import TeamMemberSelect from "./TeamMemberSelect";
+import { render, fireEvent, screen } from "@testing-library/react";
+
+let teamMembers;
+
+const handleMemberSelect = (members) => {
+  console.log("Parent Comp", members);
+};
+
```
Comment (hwilliams314) [inline web-ui/src/components/team-member/TeamMemberSelect.test.js:10]: should there be a console.log here?

File: web-ui/src/components/team-member/TeamMemberSelect.jsx (hunk 2 above)
Comment (jahanner) [reply web-ui/src/components/team-member/TeamMemberSelect.jsx:8]: Whichever component calls TeamMemberSelect will have to do the team api call and pass the members down

File: web-ui/src/components/team-member/TeamMemberSelect.jsx (hunk 3 above)
Comment (jahanner) [reply web-ui/src/components/team-member/TeamMemberSelect.jsx:19]: I know it looks a bit strange but it's basically a toggle for filtering the selected members and assigning the appropriate className

File: web-ui/src/components/team-member/TeamMemberSelect.test.js (hunk 4 above)
Comment (jahanner) [reply web-ui/src/components/team-member/TeamMemberSelect.test.js:10]: Yeah just so I can use the members arg when the test runs

File: web-ui/src/components/team-member/TeamMemberSelect.jsx (hunk 1 above)
Comment (mkimberlin) [reply web-ui/src/components/team-member/TeamMemberSelect.jsx:13]: You gonna just pretend MIKE and Mike aren't the same guy?? Come on, man...

**Pull request merged**